regolith/helpers/u_contacthelper.py

```diff
@@ -82,7 +88,11 @@ def db_updater(self):
             notes = current.get('notes', [])
             aliases = current.get('aka', [])
         else:
-            pdoc.update({"date": dt.today()})
+            if not rc.inst:
+                print("Institution (--inst) is required to create a new contact")
+                return
+            pdoc.update({"date": dt.date.today()})
+            pdoc.update({"institution": rc.inst})
             notes = []
             aliases = []
             UUID = str(uuid.uuid4())
```

Comment:
this should be lower-case for a string variable.

Reply:
changed to uniqueidentifier, it gets angry if I try to use uuid